test/integration/driver.go

```diff
@@ -125,7 +125,6 @@ func setAuth(params infra) infra {
 	out := params
 	out.Name = "(auth infra)"
 	out.Auth = proxyconfig.ProxyMeshConfig_MUTUAL_TLS
-	out.Ingress = false
```

Comment:
is this the only place that you need to test for ingress integration with auth? I vaguely remember auth being disabled in reachability tests.